src/_scss/pages/agency/index.scss

```diff
@@ -29,6 +29,71 @@
 		}
 	}
 
+    .accordion-container {
+        @media(max-width: $tablet-screen) { 
+            margin-bottom: 16px;
+        }
+    }
+
+    .accordion-container .heading {
+        height: 20px;
+        border: none;
+        color: $color-primary;
+        padding-left: 8px;
+        width: 100%;
+        padding-right: 0;
+        font-size:rem(16);
```

Comment:
This font size is 14 in the Zeplin
Also this text needs to be pushed up just a tiny bit, to be vertically centered in the toggle, like the text next to the toggle

Reply:
yeah don't know why that happens

Reply:
fixed everything up i think

Reply:
At screen sizes >1200px I think the margin-top: -2px is causing the 'what is this' text to be slightly higher than the 'view outlays' text now
And currently at <1200px the 'What is this' text moves to below the view Outlays toggle, but then at 992 it moves back over to the right side, and while its there it isn't aligned with the View Outlays toggle.
At around 435px and below the 'What is this' text wraps onto two lines

Reply:
fixed